src/monitor.cc

```diff
@@ -65,6 +69,43 @@ namespace ynode {
 // our singleton instance
 NodeMonitor* NodeMonitor::instance_ = NULL;
 
+// some utility functions to make code cleaner
+static inline v8::Local<v8::String> v8_str( const char* s ) {
+    Nan::MaybeLocal<v8::String> s_maybe = Nan::New<v8::String>(s);
+    return s_maybe.ToLocalChecked();
+}
+
+static inline v8::Local<v8::Value> getObjectProperty(const v8::Local<v8::Object>& object,
```

Comment:
It's just a (strong) suggestion (and always has been).  It's not actually a problem if it doesn't get inlined.. sometimes that would be better for code expansion problems if the compiler decides to.